client/state/selectors/index.js

```diff
@@ -45,3 +45,5 @@ export isSharingButtonsSaveSuccessful from './is-sharing-buttons-save-successful
 export isSiteBlocked from './is-site-blocked';
 export isSiteSupportingImageEditor from './is-site-supporting-image-editor';
 export isTransientMedia from './is-transient-media';
+export isRequestingReaderTeams from './is-requesting-reader-teams';
+export getReaderTeams from './get-reader-teams';
```

Comment:
I'd bet this happened because of a bad rebase, but usually we strive to follow an alphabetical order when listing selector exports here, so these two would probably have to be moved.